schema.json

```diff
@@ -1,8 +1,18 @@
 {
-    "id": "http://json-schema.org/draft/schema#",
-    "$schema": "http://json-schema.org/draft/schema#",
-    "description": "Core schema meta-schema",
+    "$schema": "http://json-schema.org/draft/schema",
+    "id": "http://json-schema.org/draft/schema",
+    "title": "Core schema meta-schema",
     "definitions": {
+        "jsonReference": {
+            "type": "object",
+            "properties": {
+                "$ref": {
+                    "type": "string",
+                    "format": "uriref"
+                }
+            },
+            "required": [ "$ref" ]
+        },
         "schemaArray": {
             "type": "array",
             "minItems": 1,
```

Comment:
Not quite sure I follow but I need to re-do this whole series of commits anyway so I will double check as I do that.